src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
`tabIndex` is the computed index, which Edge/UWP reports to be 0 for divs. `getAttribute('tabIndex')` will pull the index if it's been manually defined on the element. If it's been defined to 0, that indicates that the consumer wants a (presumably normally unfocusable element) to be focusable.